lib/perf_hooks.js

```diff
@@ -30,6 +33,14 @@ const {
   NODE_PERFORMANCE_MILESTONE_ENVIRONMENT
 } = constants;
 
+const { Buffer } = require('buffer');
+
+const kRunning = Symbol('delay-timer-running');
```

Comment:
Can we use a greppable description as suggested in https://github.com/nodejs/node/blob/master/doc/guides/using-symbols.md, e.g.

```suggestion
const kRunning = Symbol('kRunning');
```

or (I’d personally prefer)

```suggestion
const kDelayTimerRunning = Symbol('kDelayTimerRunning');
```